yarn-project/simulator/src/public/index.test.ts

```diff
@@ -731,42 +731,43 @@ describe('ACIR public execution simulator', () => {
     });
   });
 
-  describe('Historical header in public context', () => {
-    let contractAddress: AztecAddress;
-    let callContext: CallContext;
-    let assertHeaderPublicArtifact: FunctionArtifact;
-    let functionData: FunctionData;
-
-    beforeAll(() => {
-      contractAddress = AztecAddress.random();
-      callContext = makeCallContext(AztecAddress.random());
-      assertHeaderPublicArtifact = TestContractArtifact.functions.find(f => f.name === 'assert_header_public')!;
-      functionData = FunctionData.fromAbi(assertHeaderPublicArtifact);
-    });
-
-    beforeEach(() => {
-      publicContracts.getBytecode.mockResolvedValue(assertHeaderPublicArtifact.bytecode);
-    });
-
-    it('Header is correctly set', () => {
-      const args = encodeArguments(assertHeaderPublicArtifact, [header.hash()]);
-
-      const execution: PublicExecution = { contractAddress, functionData, args, callContext };
-      executor = new PublicExecutor(publicState, publicContracts, commitmentsDb, header);
-
-      expect(() => simulate(execution, globalVariables)).not.toThrow();
-    });
-
-    it('Throws when header is not as expected', async () => {
-      const unexpectedHeaderHash = Fr.random();
-      const args = encodeArguments(assertHeaderPublicArtifact, [unexpectedHeaderHash]);
-
-      const execution: PublicExecution = { contractAddress, functionData, args, callContext };
-      executor = new PublicExecutor(publicState, publicContracts, commitmentsDb, header);
-
-      const { revertReason, reverted } = await simulate(execution, globalVariables);
-      expect(reverted).toBe(true);
-      expect(revertReason?.message).toMatch(`Invalid header hash`);
-    });
-  });
+  // TODO(4840): add AVM opcodes for getting header (members)
```

Comment:
Would a describe.skip work?
Otherwise, /* */ might be less disruptive?
(I don't feel strongly on any changes tho)

Reply:
Good point! I didn't think .skip would work, but I was thinking of e2e stuff when we call functions by name, vs here we are looking for a string in the artifact. I'll do .skip :)